roles/nexus/defaults/main.yml

```diff
@@ -64,6 +64,8 @@ nexus_proxy_exclude_hosts:
   - "localhost"
   - "127.*"
   - "[::1]"
+nexus_security_ssl_url: "/service/rest/v1/security/ssl"
+nexus_security_truststore_url: "/service/rest/v1/security/ssl/truststore"
```

Comment:
In which scenario would these need to be changed?

Or phrased differently: Do these really need to be role variables?

Reply:
do you mean, it could be hardcoded?

Reply:
and I do not know, if there will be scenarios